app/src/main/res/layout/activity_player.xml

```diff
@@ -38,5 +38,20 @@
             android:visibility="gone"
             tools:ignore="KeyboardInaccessibleWidget"
             tools:visibility="visible" />
+
+        <androidx.appcompat.widget.AppCompatImageButton
+            android:id="@+id/unlock_screen_button"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_gravity="start|top"
+            android:layout_marginStart="@dimen/exo_player_controls_display_padding"
+            android:layout_marginTop="48dp"
+            android:layout_marginEnd="@dimen/exo_player_controls_display_padding"
+            android:background="@drawable/ic_screen_unlock_white_24dp"
```

Comment:
You might want to add an actual background to ensure the button is always visible above the movie content, maybe a scrim/shadow, or a semi-transparent circle/rounded square.